Show test-output on failures more prominently in Jenkins

In the following CI result, the test output simply states that the VM exits with status code 1:

https://jenkins.cilium.io/job/Cilium-PR-Ginkgo-Tests-Validated/3306/

Test result page:
```
runtime.RuntimeValidatedPolicyEnforcement Policy Enforcement Default Default values (from (runtime.xml))

Failing for the past 3 builds (Since ￼#3304 )
Took 20 sec.
Stacktrace
/home/jenkins/workspace/Cilium-PR-Ginkgo-Tests-Validated/src/github.com/cilium/cilium/test/ginkgo-ext/scopes.go:295
error starting VM "runtime": exit status 1
/home/jenkins/workspace/Cilium-PR-Ginkgo-Tests-Validated/src/github.com/cilium/cilium/test/test_suite_test.go:157
```

Console output page:
```
21:28:16 [Cilium-PR-Ginkgo-Tests-Validated] Running shell script
21:28:16 + cd /home/jenkins/workspace/Cilium-PR-Ginkgo-Tests-Validated/src/github.com/cilium/cilium/test
21:28:16 + ginkgo --focus= RuntimeValidated* -v --failFast=true
21:29:14 level=info msg="environment variable \"K8S_VERSION\" was not set; setting to default value \"1.10\""
21:29:14 Running Suite: runtime
21:29:14 ======================
21:29:14 Random Seed: 1526592496
21:29:14 Will run 58 of 285 specs
21:29:14 
21:29:14 SSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS
21:29:14 ------------------------------
21:29:14 RuntimeValidatedPolicyEnforcement Policy Enforcement Default 
21:29:14   Default values
21:29:14   /home/jenkins/workspace/Cilium-PR-Ginkgo-Tests-Validated/src/github.com/cilium/cilium/test/ginkgo-ext/scopes.go:338
21:29:30 
21:29:30 • Failure in Spec Setup (BeforeEach) [20.872 seconds]
21:29:30 RuntimeValidatedPolicyEnforcement [BeforeEach]
21:29:30 /home/jenkins/workspace/Cilium-PR-Ginkgo-Tests-Validated/src/github.com/cilium/cilium/test/ginkgo-ext/scopes.go:306
21:29:30   Policy Enforcement Default
21:29:30   /home/jenkins/workspace/Cilium-PR-Ginkgo-Tests-Validated/src/github.com/cilium/cilium/test/ginkgo-ext/scopes.go:306
21:29:30     Default values
21:29:30     /home/jenkins/workspace/Cilium-PR-Ginkgo-Tests-Validated/src/github.com/cilium/cilium/test/ginkgo-ext/scopes.go:338
21:29:30 
21:29:30     error starting VM "runtime": exit status 1
21:29:30 
21:29:30     /home/jenkins/workspace/Cilium-PR-Ginkgo-Tests-Validated/src/github.com/cilium/cilium/test/test_suite_test.go:157
21:29:30 ------------------------------
21:29:30 SSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS
21:29:30 
21:29:30 Summarizing 1 Failure:
21:29:30 
21:29:30 [Fail] [BeforeEach] RuntimeValidatedPolicyEnforcement Policy Enforcement Default Default values 
21:29:30 /home/jenkins/workspace/Cilium-PR-Ginkgo-Tests-Validated/src/github.com/cilium/cilium/test/test_suite_test.go:157
21:29:30 
21:29:30 Ran 1 of 285 Specs in 20.873 seconds
21:29:30 FAIL! -- 0 Passed | 1 Failed | 0 Pending | 284 Skipped --- FAIL: TestTest (20.88s)
21:29:30 FAIL
21:29:30 
21:29:30 Ginkgo ran 1 suite in 1m13.476813534s
21:29:30 Test Suite Failed
```

Inside the test artifacts zip, there is only two files, empty `cmds.log` and the `test-output.log` that finally actually points out some useful information:

```
time="2018-05-17T21:29:08Z" level=info msg="Vagrant: running command 'vagrant status runtime --machine-readable'"
time="2018-05-17T21:29:09Z" level=info msg="Vagrant:Create: running 'vagrant provision runtime'"
time="2018-05-17T21:29:09Z" level=info msg="Vagrant: running command 'vagrant provision runtime'"
==> runtime: Running provisioner: shell...
    runtime: Running: inline script
==> runtime: Running provisioner: file...
==> runtime: Running provisioner: shell...
    runtime: Running: /tmp/vagrant-shell20180517-32465-157jyaw.sh
    runtime: editing journald configuration
    runtime: restarting systemd-journald
    runtime: getting status of systemd-journald
    runtime: * systemd-journald.service - Journal Service
    runtime:    Loaded: loaded (/lib/systemd/system/systemd-journald.service; static; vendor preset: enabled)
    runtime:    Active: active (running) since Thu 2018-05-17 21:29:12 UTC; 19ms ago
    runtime:      Docs: man:systemd-journald.service(8)
    runtime:            man:journald.conf(5)
    runtime:  Main PID: 2984 (systemd-journal)
    runtime:    Status: "Processing requests..."
    runtime:     Tasks: 1 (limit: 4915)
    runtime:    Memory: 788.0K
    runtime:       CPU: 5ms
    runtime:    CGroup: /system.slice/systemd-journald.service
    runtime:            `-2984 /lib/systemd/systemd-journald
    runtime: 
    runtime: May 17 21:29:12 runtime systemd-journald[2984]: Journal started
    runtime: May 17 21:29:12 runtime systemd-journald[2984]: Runtime journal (/run/log/journal/5bc2a0293ab24ede8e038912c3e65019) is 3.7M, max 30.0M, 26.3M free.
    runtime: done configuring journald
    runtime: HEAD is now at 930a67c... Fix pattern rule in repeated and map field (#58)
    runtime: # generates the PGV binary and installs it into OPATH/bin
    runtime: go install .
    runtime: updating /etc/resolv.conf
    runtime: can't load package: package github.com/cilium/cilium/test/bpf: C source files not allowed when not using cgo or SWIG: bpf-event-test.c unit-test.c
    runtime: go tool vet envoy plugins bpf cilium daemon monitor cilium-health bugtool
    runtime: contrib/scripts/check-fmt.sh
    runtime: Unformatted Go source code:
    runtime: ./test/helpers/kubectl.go
    runtime: diff -u ./test/helpers/kubectl.go.orig ./test/helpers/kubectl.go
    runtime: --- ./test/helpers/kubectl.go.orig	2018-05-17 21:29:27.956411309 +0000
    runtime: +++ ./test/helpers/kubectl.go	2018-05-17 21:29:27.956411309 +0000
    runtime: @@ -30,8 +30,8 @@
    runtime:  
    runtime:  	"github.com/cilium/cilium/api/v1/models"
    runtime:  	"github.com/cilium/cilium/pkg/annotation"
    runtime: -	"github.com/cilium/cilium/test/config"
    runtime:  	cnpv2 "github.com/cilium/cilium/pkg/k8s/apis/cilium.io/v2"
    runtime: +	"github.com/cilium/cilium/test/config"
    runtime:  	ginkgoext "github.com/cilium/cilium/test/ginkgo-ext"
    runtime:  
    runtime:  	"github.com/asaskevich/govalidator"
    runtime: Makefile:212: recipe for target 'precheck' failed
    runtime: make: *** [precheck] Error 1
The SSH command responded with a non-zero exit status. Vagrant
assumes that this means the command failed. The output for this command
should be in the log above. Please read the output to determine what
went wrong.
time="2018-05-17T21:29:29Z" level=error msg="Error starting VM" error="exit status 1"
time="2018-05-17T21:29:29Z" level=error msg="Error starting VM" error="exit status 1"
```

It would be nice to get the tail of this `test-output.log` printed somewhere - either in the test result page or in the console logs when the testsuite fails. This would simplify seeing what kind of error occurred.

--- a/test/helpers/vagrant.go
+++ b/test/helpers/vagrant.go
@@ -22,8 +22,9 @@ import (
 	"strings"
 
 	"github.com/cilium/cilium/test/config"
-	ginkgo "github.com/cilium/cilium/test/ginkgo-ext"
+	ginkgoext "github.com/cilium/cilium/test/ginkgo-ext"
 
+	"github.com/onsi/ginkgo"
 	"github.com/sirupsen/logrus"
 )
 
@@ -55,6 +56,9 @@ func CreateVM(scope string) error {
 	if err != nil {
 		return fmt.Errorf("error getting stderr: %s", err)
 	}
+
+	go io.Copy(ginkgoext.GinkgoWriter, stderr)
+	go io.Copy(ginkgoext.GinkgoWriter, stdout)
 
 	go io.Copy(ginkgo.GinkgoWriter, stderr)
 	go io.Copy(ginkgo.GinkgoWriter, stdout)

